### PR TITLE
Adding csvs for recent Pioneer glider recoveries

### PR DESCRIPTION
**Pull request by cdobs**
Adds recovered ingestion csvs:
-CP05MOAS-GL379_R00012
-CP05MOAS-GL388_R00011
-CP05MOAS-GL559_R00002
-CP05MOAS-PG583_R00004

### CODE REVIEW

File: CP05MOAS-GL379/CP05MOAS-GL379_R00012_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00012/merged/cp_*.mrg,CP05MOAS-GL379-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00012/dvl/*.PD0,CP05MOAS-GL379-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00012/merged/cp_*.mrg,CP05MOAS-GL379-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00012/merged/cp_*.mrg,CP05MOAS-GL379-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00012/merged/cp_*.mrg,CP05MOAS-GL379-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/R00012/merged/cp_*.mrg,CP05MOAS-GL379-05-PARADM000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-GL379/CP05MOAS-GL379_R00012_ingest.csv:7]: Looks good.

File: CP05MOAS-GL388/CP05MOAS-GL388_R00011_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00011/merged/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00011/dvl/*.PD0,CP05MOAS-GL388-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00011/merged/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00011/merged/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00011/merged/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00011/merged/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-GL388/CP05MOAS-GL388_R00011_ingest.csv:7]: Looks good.

File: CP05MOAS-GL559/CP05MOAS-GL559_R00002_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/R00002/merged/cp_*.mrg,CP05MOAS-GL559-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/R00002/dvl/*.PD0,CP05MOAS-GL559-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/R00002/merged/cp_*.mrg,CP05MOAS-GL559-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/R00002/merged/cp_*.mrg,CP05MOAS-GL559-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/R00002/merged/cp_*.mrg,CP05MOAS-GL559-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/R00002/merged/cp_*.mrg,CP05MOAS-GL559-05-PARADM000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-GL559/CP05MOAS-GL559_R00002_ingest.csv:7]: Looks good.

File: CP05MOAS-PG583/CP05MOAS-PG583_R00004_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-01-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-02-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-03-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-04-FLORTO000,recovered_host,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-05-NUTNRM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-PG583/R00004/merged/cp_*.mrg,CP05MOAS-PG583-06-PARADM000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-PG583/CP05MOAS-PG583_R00004_ingest.csv:8]: Looks good.  The NUTNR parser format is different than the others, but matches past ingestion CSVs.

**Review by snwhite — APPROVED** [review]

File: CP05MOAS-GL379/CP05MOAS-GL379_R00012_ingest.csv (hunk 1 above)
Comment (IDCalTech) [reply CP05MOAS-GL379/CP05MOAS-GL379_R00012_ingest.csv:7]: Looks good.

File: CP05MOAS-PG583/CP05MOAS-PG583_R00004_ingest.csv (hunk 4 above)
Comment (IDCalTech) [reply CP05MOAS-PG583/CP05MOAS-PG583_R00004_ingest.csv:8]: Looks good

File: CP05MOAS-GL388/CP05MOAS-GL388_R00011_ingest.csv (hunk 2 above)
Comment (IDCalTech) [reply CP05MOAS-GL388/CP05MOAS-GL388_R00011_ingest.csv:7]: Looks good

File: CP05MOAS-GL559/CP05MOAS-GL559_R00002_ingest.csv (hunk 3 above)
Comment (IDCalTech) [reply CP05MOAS-GL559/CP05MOAS-GL559_R00002_ingest.csv:7]: Looks good

**Review by IDCalTech — APPROVED** [review]
All files look good

**Pull request merged**